integration-tests/lib/requestData.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  generateRandomNumber,
+  generateZendeskRequestDate
+} from '../utils/helpers'
+
+const validRequestData = {
+  request: {
+    subject: `Integration Test Request - ` + generateRandomNumber(),
+    ticket_form_id: 5603412248860,
+    custom_fields: [
+      {
+        id: 5605352623260,
```

Comment:
At least adding some constants for all these "magic numbers" would make it clearer what they're all for.
That way, if they ever change, it'll be a lot easier to see what they're for. Just name the variables with something like the name they're labelled with in Zendesk.

Reply:
Done.